views/group/new.pug

```diff
@@ -40,7 +38,7 @@ block content
       div(id="otherDiv" class=`${(isEditing && place) ? '' : 'hidden'}`)
         label(class='text-lg font-bold' for='otherInput') Más hely
         input(type="text", name="place", id="otherInput", maxlength="100", class="w-full px-3 py-2 rounded form-element", value=`${(isEditing && place) ? place : ''}`)
-          //- button(type="button", class="px-2 py-1 border border-gray-600 rounded-md hover:border-gray-500", @click="open = true") Naptár
+      button(type="button" class="self-start btn btn-meeting-default animate-hover" @click="open = true") Naptár
```

Comment:
I guess this button needs to move up a few lines to be under `div(id = "floorDiv"...` so that it appears only when you are choosing a floor (and should be available when choosing link or other place)